packages/itwinui-css/backstop/scenarios/slider.js

```diff
@@ -6,27 +6,25 @@ const { scenario, hover, focus } = require('../scenarioHelper');
 
 module.exports = [
   // Slider types
-  scenario('Type Default', {
-    selectors: ['#demo-default'],
+  scenario('Type Horizontal', {
```

Comment:
could you do this change in two commits and link to them so it's easy to compare what changed visually?

right now the diff is unhelpful
<img width="1112" alt="" src="https://github.com/iTwin/iTwinUI/assets/9084735/e81c5223-6e6b-4d31-b179-3b3666a73026">

Reply:
The demos I have currently aren't 1-to-1 with what they were previously.  Previously all demos were wrapped with `.iui-input-container`.  Do you want me to try to match the demos with what they looked like previously?

Reply:
would it be too much effort to change the demo structure in a separate preceding PR? mainly i would like git to show a side-by-side diff rather than separate deleted+add files

Reply:
I've made all the tests match how they looked previously & discovered a bug for the vertical slider along the way.